routers/auth.py

```
@@ -0,0 +1,53 @@
+def signup(payload: UserCreate):
+    users = read_json("users.json", default=[])
+    if users is None:
+        users = []
+    if any(user["email"] == payload.email for user in users):
+        raise HTTPException(status_code=400, detail="Email already registered")
+    user = {
+        "id": next_id(users),
+        "email": payload.email,
+        "nickname": payload.nickname,
+        "hashed_password": hash_password(payload.password),
+        "created_at": datetime.now().isoformat()
```

Ruff (0.14.13)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)